kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/CreateLearnerAccountForm.vue

```diff
@@ -63,6 +63,14 @@
         return { type: 'BACK', value: Boolean(this.setting) };
       },
     },
+    beforeRouteLeave(to, from, next) {
+      if (!this.$store.state.wizardNavigated) {
+        if (from.meta.previousNavigationAllowed.includes(to.name)) {
+          return this.backEvent;
+          // return this.wizardService.send({ type: 'BACK' });
```

Comment:
I wonder if these two lines ought to be combined so the service is sent the `backEvent`?